plugins/mkt-queue-sync/src/classes/Synchronizer.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MikrotikQueueSync;
+
+use Ubnt\UcrmPluginSdk\Security\PermissionNames;
+use Ubnt\UcrmPluginSdk\Service\UcrmApi;
+use Ubnt\UcrmPluginSdk\Service\UcrmOptionsManager;
+use Ubnt\UcrmPluginSdk\Service\PluginConfigManager;
+use Ubnt\UcrmPluginSdk\Service\PluginLogManager;
+//use Ubnt\UcrmPluginSdk\Service\UcrmSecurity;
+//use MikrotikQueueSync\Service\Logger;
+//use MikrotikQueueSync\Service\OptionsManager;
+//use MikrotikQueueSync\Service\UcrmApi;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$GLOBALS['sumDownloadLimitAt'] = 0;
+$GLOBALS['sumUploadLimitAt'] = 0;
+$GLOBALS['sumDownloadVendido'] = 0;
+$GLOBALS['sumUploadVendido'] = 0;
+
+class Synchronizer
+{
+    /**
+     * @var OptionsManager
+     */
+   // private $optionsManager;
+
+    /**
+     * @var Logger
+     */
+    //private $logger;
+
+    /**
+     * @var UcrmApi
+     */
+    //private $ucrmApi;
+
+    /**
+     * @var \RouterosAPI
+     */
+    //private $routerosAPI;
+
+	
+
+    public function __construct(
```

Comment:
Just for cleaner code. Remove constructor, commented properties of this class and `use` statements .

Reply:
Hi @janprochazkacz  code cleaned... Just i don't understand completely what do your refer with "use statements" ... Sorry my coding knowledge is very basic.